crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -1311,6 +1320,9 @@ where
         .expect("Must always be able to get chain constants")
         .confirmation_depth_k();
 
+    // TODO: Probably should have public parameters in chain constants instead
+    let kzg = Kzg::new(kzg::test_public_parameters());
```

Comment:
It should be an argument so it can be customized (eventually), the question is where it comes from. Not sure what you meant in that comment though.

Reply:
Like having a global static:
```
pub static KZG: Lazy<Kzg> = Lazy::new(|| Kzg::new(kzg::test_public_parameters()));
```

Which we can access or just pass reference to everywhere

Reply:
Well, if we pass the reference it doesn't matter whether we get it from here or from a static, while global variables I'd prefer to avoid. It isn't particularly expensive to create an instance of Kzg, the TODO is about moving `PublicParameters` in chain constants, we have such data structure with a few properties, so we can potentially have different chains with different parameters.